tools/compiler/src/parser.rs

```diff
@@ -31,41 +31,14 @@ pub struct ChainConfig {
     pub native_assets: Vec<Metadata>,
     pub canonical_numeraires: Vec<String>,
     pub priority_scores_by_base: HashMap<BaseDenom, u64>,
+    pub badges_by_base: HashMap<BaseDenom, Vec<AssetImage>>,
 }
 
 #[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
 pub struct EntityMetadata {
     pub name: String,
     pub url: String,
-    pub images: Vec<Image>,
-}
-
-#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
-pub struct Image {
-    #[serde(skip_serializing_if = "Option::is_none")]
-    pub png: Option<String>,
-    #[serde(skip_serializing_if = "Option::is_none")]
-    pub svg: Option<String>,
-}
-
-impl From<Image> for AssetImage {
-    fn from(image: Image) -> Self {
-        AssetImage {
-            png: image.png.unwrap_or_default(),
-            svg: image.svg.unwrap_or_default(),
-            theme: None,
-        }
-    }
-}
-
-pub trait IntoPbImages {
-    fn into_pb_images(self) -> Vec<AssetImage>;
-}
-
-impl IntoPbImages for Vec<Image> {
-    fn into_pb_images(self) -> Vec<AssetImage> {
-        self.into_iter().map(AssetImage::from).collect()
-    }
```

Comment:
Found defining a local image type to be unnecessary when we can just import the protobuf type directly